### PR TITLE
test_open_pr

### PR DESCRIPTION
**Pull request by mozart0**
test_open_pr_msg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # hello
-Hello world
+Hello world!
```
Comment (hbhftgf) [inline README.md:2]: single_comment_on_file edit

**Comment by hbhftgf** [comment]
single_comment_in_conv

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # hello
```
Comment (hbhftgf) [inline README.md]: comment_1_of_thread

File: README.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
...
-Hello world
```
Comment (hbhftgf) [inline README.md:2]: comment_2_of_thread

**Review by hbhftgf — COMMENTED** [review]
submit_thread_comments

File: README.md (hunk 1 above)
Comment (hbhftgf) [inline README.md:2]: comment_of_thread

**Review by hbhftgf — APPROVED** [review]
approve_comment

File: README.md (hunk 2 above)
Comment (hbhftgf) [inline README.md]: comment_of_thread_again

**Review by hbhftgf — CHANGES_REQUESTED** [review]
request_change_comment

File: README.md (hunk 1 above)
Comment (mozart0) [reply README.md:2]: you are right

File: README.md (hunk 2 above)
Comment (mozart0) [reply README.md]: replay_1

File: README.md (hunk 3 above)
Comment (mozart0) [reply README.md:2]: reply_2

**Comment by mozart0** [comment]
author_comment

**Comment by mozart0** [comment]
author_comment_and_close

**Comment by mozart0** [comment]
author_comment_and_reopen

**Comment by hbhftgf** [comment]
> author_comment_and_reopen

owner_quote_replay

**Pull request merged**